Maple2.Model/Game/Party/PartyMember.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Numerics;
+using System.Threading;
+using Maple2.Model.Common;
+using Maple2.PacketLib.Tools;
+using Maple2.Tools;
+using Maple2.Tools.Extensions;
+
+namespace Maple2.Model.Game;
+
+public class PartyMember : IByteSerializable, IDisposable {
+    public long PartyId { get; init; }
+    public required PlayerInfo Info;
+    public long JoinTime;
+    public long LoginTime;
+    public long AccountId => Info.AccountId;
+    public long CharacterId => Info.CharacterId;
+    public string Name => Info.Name;
+    public byte ReadyState = 0;
+
+    public CancellationTokenSource? TokenSource;
+
+    public void WriteTo(IByteWriter writer) {
+        writer.WriteLong(Info.AccountId);
+        writer.WriteLong(Info.CharacterId);
+        writer.WriteUnicodeString(Info.Name);
```

Comment:
shouldn't we just be using Character? I believe it's the same struct.

Reply:
+1, use PlayerInfo if you can. It has less data so it will be less stale, and there's infrastructure to keep it updated.